db/models.py

```diff
@@ -1 +1,29 @@
+from tkinter.constants import CASCADE
+
 from django.db import models
+from django.db.models import SET_NULL
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
The `on_delete=models.CASCADE` is used correctly here, as it implies that a Skill cannot exist without a Race.